packages/playground/nested-deps/vite.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
       'test-package-b',
       'test-package-c',
       'test-package-c/side',
-      'test-package-d    > test-package-d-nested'
+      'test-package-d > test-package-d-nested'
```

Comment:
This is definitely not intended!
The test explicitly test multiple space case

Reply:
reverted in 41356c52